transformations/GermanGenderSwap/transformation.py

```diff
@@ -0,0 +1,2433 @@
+# -*- coding: utf-8 -*-
+
+import random
+
+from typing import List
+from checklist.editor import Editor
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+#TOP 1000 MOST COMMON GERMAN MALE AND FEMALE NAMES,
```

Comment:
The # in a comment should always be followed by a space. Would you mind running a pep8 linter over your code and correcting the found stylistic inaccuracies?